### PR TITLE
Mayhem Integration

### PR DESCRIPTION
**Pull request by prateekvishnu**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Namespaced project name that the target belongs to
+project: apollo-rs
+
+# Target name (should be unique within the project)
+target: apollo-rs
+
+# Base image to run the binary in.
+image: ghcr.io/prateekvishnu/apollo-rs:latest
```
Comment (xansec) [inline Mayhemfile:8]: Please omit the image name in the Mayhemfile.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Namespaced project name that the target belongs to
+project: apollo-rs
+
+# Target name (should be unique within the project)
+target: apollo-rs
```
Comment (xansec) [inline Mayhemfile:5]: Your Mayhemfile has two targets. Please either remove one of the `cmd:` fields, or split this into multiple Mayhemfiles (and actions). The target name should also reflect the name of the target binary. So, for example, if your target command was `/lexer`, the target field here should show `lexer` (this helps disambiguate target runs if, for example, you do have multiple Mayhemfiles for the same project).

**Review by xansec — CHANGES_REQUESTED** [review]

**Comment by prateekvishnu** [comment]
Resolved changes.

**Review by xansec — APPROVED** [review]

**Comment by xansec** [comment]
LGTM!

**Pull request merged**